lib/functions/last.js

```diff
@@ -0,0 +1,22 @@
+const { filter } = require("./filter");
+
+const last = (iterable, filterCondition = () => true) => {
+  const filtered = filter(iterable, filterCondition);
+  const iterator = filtered[Symbol.iterator]();
+
+  let lastValue;
+  for (;;) {
+    const { done, value } = iterator.next();
+    if (done) {
+      break;
+    }
+    lastValue = value;
+  }
+  if (!lastValue) {
```

Comment:
Instead of doing this check, I'd suggest using a count or a flag, since the lastValue can be by itself `null`or `undefined`, `""`, etc.